lib/generators/setup_generator.rb

```diff
@@ -71,6 +71,13 @@ def copy_configs
     answer = prompt_for_value("Project Subtitle (Header <h2>)", "Template Subtitle")
     config_set("public", "project_subtitle", answer)
 
+    answer = prompt_for_value("Primary Language", "en")
+    config_set("public", "language_default", answer)
+
+    answer = prompt_for_value("All Languages (separate with a pipe: en|es|de)", "en")
+    puts "ANSWER: #{answer}"
```

Comment:
Oh, I think I just forgot to revisit that part after I was trying a few things out.  Originally the config file had a commented line like `# languages: en|es` to demonstrate multiple languages without actually shipping them into the new app that way, so I was trying to select the second instance of it with spaces at the beginning instead of a `#`.  I ended up just putting the `en|es` explanation elsewhere and removing the extra commented line, so I can change this back to no spaces, I believe :)